core/server/models/member-status-event.js

```diff
@@ -17,15 +17,12 @@ const MemberStatusEvent = ghostBookshelf.Model.extend({
             return qb.clear('select')
```

Comment:
I would also be interested in a double check of my logic for updating the handling of `member_status_events`